raiden/routing.py

```diff
@@ -108,9 +108,10 @@ def get_best_routes(
         to_address,
     )
 
-    log.warn(
-        'No routes available from %s to %s' % (pex(from_address), pex(to_address))
-    )
+    if not neighbors_heap:
```

Comment:
LOL we had this warning always on before!??!

Reply:
Yeah, but only since the structlog refactor. Must have accidentally fallen victim to the “remove logging conditionals” effort

Reply:
Perhaps we should do a re-check of that PR to see if other **needed** conditionals were removed.

Reply:
I already checked it when I found this one. This was the only one I could find.